checks/plugins.py

```diff
@@ -36,10 +36,11 @@ def checkPluginList(lines):
         thirdPartyPlugins = []
 
         for s in pluginList:
-            if ('     ' in s):
-                pluginFormatter = s.split(' ')
-                pluginFormatter = pluginFormatter[-1].split('.')
-                thirdPartyPlugins.append(pluginFormatter[0])
+            if '     ' in s:
+                timestamp, plugin = s[0:12], s[16:]
```

Comment:
In some cases, log lines on Linux can begin with a loglevel indicator like `info:` instead of a timestamp. It would probably be safer to find the first `:` and split on that.

Reply:
Updated to now split using `: ` with the whitespace after it as timestamps have multiple.